src/build.rs

```diff
@@ -105,7 +105,7 @@ pub fn build(
 
 	if !skip_optimization {
 		let preserved_exports = match target_runtime {
-			TargetRuntime::PWasm(_) => vec![target_runtime.symbols().call],
```

Comment:
It should not work like this

With pwasm, first optimisation (this one) is optimising everything that is not called from `call` method, not `deploy`

The second optimisation, which is done in `pack_instance` optimises everything that is not called from `deploy` method

Reply:
hmm ok, this is actually not true ^^^, this is second optimisation actually

in #ec206fca6428d64b724a8668b431977f833e3fef there was a bug introduced